rules/windows/process_creation/proc_creation_win_hktl_wsass.yml

```diff
@@ -0,0 +1,28 @@
+title: HackTool - WSASS Execution for PPL Bypass and LSASS Dumping
+id: 589ac73f-8e12-409c-964e-31a2f5775ae2
+status: experimental
+description: Detects execution of WSASS, a tool used to dump LSASS memory on Windows systems by leveraging Windows Error Reporting (WER)'s WerFaultSecure.EXE to bypass Protected Process Light (PPL) protections.
+references:
+    - https://github.com/TwoSevenOneT/WSASS
+    - https://www.zerosalarium.com/2025/09/Dumping-LSASS-With-WER-On-Modern-Windows-11.html
+author: Swachchhanda Shrawan Poudel (Nextron Systems)
+date: 2025-10-21
+tags:
+    - attack.credential-access
+    - attack.t1003.001
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_img:
+        Image|endswith: '\wsass.exe'
+    selection_hash:
+        Hashes|contains: 'IMPHASH=32F5095C9BBDCACF28FD4060EB4DFC42'
+    selection_cli:
+        # change to |re|i after Sigma v2.0 release
+        CommandLine|re: (?i)\.exe[\"\']?\s+[\"\']?\S{,64}\s+\d{,5}  # wsass.exe "path to werfaultsecure" lsass_pid
```

Comment:
```suggestion
        CommandLine|re: '(?i)\.exe[\"\']?\s+[\"\']?\S{,64}\s+\d{,5}'  # wsass.exe "path to werfaultsecure" lsass_pid
```

Reply:
ah yes makes sense. Maybe also add a comment for this :)

Reply:
we usually want to encapsulate values with ' to make whitespace explicit. To prevent future confusion/changes we should document/comment why not doing so here is needed here